docs/testplan/snappi-tests/capacity_test.md

```diff
@@ -0,0 +1,91 @@
+# SONiC Switch Capacity Test
+
+
+- [SONiC Switch Capacity Test](#sonic-switch-capacity-test)
+  - [Test Objective](#test-objective)
+  - [Test Setup](#test-setup)
+  - [Test Steps](#test-steps)
+  - [Label Structure](#label-structure)
+    - [Common labels for all metrics](#common-labels-for-all-metrics)
+    - [Metric labels](#metric-labels)
+      - [PSU Metrics](#psu-metrics)
+      - [Sensor Temperature Metrics](#sensor-temperature-metrics)
+      - [Queue Metrics](#queue-metrics)
+      - [Interface Metrics](#interface-metrics)
+
+## Test Objective
+
+This test aims to assess the true capacities of SONiC switches.
+
+## Test Setup
+
+The test is designed to be topology-agnostic, meaning it does not assume or impose a specific setup wiring, settings, or configuration. The running script will track power usage, temperature, queue watermark, and health state of all SONiC switches in a testbed.
+
```

Comment:
In this case, we'd better to use the first half ports as tx port, and the last half as rx port.